### DISCUSSION TITLE
No Way(?) To Disable Unnecessary Error (Which Should be a Warning?)

### DISCUSSION THREAD

**Discussion by machineghost** [avajs/ava#3090] [Q&A]
I have some files with old tests from my previous test runner (Mocha).  I have them completely commented out, but when I run Ava I get a ton of:

>  ✖ No tests found in src/foo.spec.js, make sure to import "ava" at the top of your test file

Is there any sort of command line flag I can provide to tell Ava "yes I know those files have no tests, I'll fix them later, but for now please stop telling me?"  It seems like that would be handy, as commented-out test files aren't actually harmful.

Also, regardless of the above, why are these errors (with red coloring and an X): if nothing is broken, shouldn't they be warnings instead?

**Comment by machineghost** [comment]
Nevermind, found `--failWithoutAssertions`!

**Comment by machineghost** [comment]
Actually, that didn't help :( Re-opening.

**Comment by novemberborn** [comment]
AVA considers it a problem if a test file (as determined by the, possibly default, glob patterns) does not contain any tests.

You could perhaps rename the file to `foo.spec.skip.js` or something similar that a negated glob pattern can exclude from the tests. Or import AVA and place `test.todo('uncomment tests in this file')` at the top.

**Comment by machineghost** [comment]
Thanks.  I still think `--failWithoutTests=false` option (or just making the existing `--failWithoutAssertions` option include such functionality) would make sense, but that's a good workaround.
